### ISSUE TITLE
[Edge] Login button redirects to regular tab, does not authenticate for popup

### ISSUE THREAD

**Issue by coldacid** [gab-ai-inc/gab-dissenter-extension#7]
# What I Did

1. Click Dissenter extension button next to address bar
2. Click "Log In" button in Dissenter popup window
3. Log in in new tab opened up
4. Click Dissenter extension button again to re-open popup

# What I Expected

* Dissenter page for current tab shown in popup

# What I Got

* Dissenter extension login page

# Browser, OS, and Extension Info

* Microsoft Edge 44.17763.1.0 (Microsoft EdgeHTML 18.17763)
* Windows 10 version 1809 (OS Build 17763.316)
* Dissenter extension v0.1.1

**Comment by mgabdev** [comment]
@coldacid thanks for submitting an issue. This is expected behavior. The reason for this is when attempting to log into Gab using the Gab.com API via Dissenter.com _through_ the extension, it is not secure and it cannot perform the OAuth redirect as the extension is simply an iFrame of https://dissenter.com.

Does the extension properly work after logging in via the newly created tab as you described?

**Comment by coldacid** [comment]
@mgabdev no, that's what I'm saying. After logging in, I still just get the login page in the popup, and there's no sign of Dissenter itself on any of my other tabs.

**Comment by mgabdev** [comment]
@coldacid Are you still experiencing this issue with the latest release of r9? https://github.com/gab-ai-inc/gab-dissenter-extension/releases/tag/r9

**Comment by coldacid** [comment]
@mgabdev let me set it up and I'll tell you.

**Comment by coldacid** [comment]
Yep, still experiencing it unfortunately.

**Comment by marknoble** [comment]
Still seeing this in the r12 release.  I was logged into Gab, but clicking the Dissenter extension results in a window with a spinner.

The DevTools Console shows the following error:
```
Uncaught ReferenceError: browser is not defined
    at Popup.e.init (popup.js:4)
    at HTMLDocument.<anonymous> (popup.js:4)
```

![Dissenter Screenshot 2021-02-16 191123](https://user-images.githubusercontent.com/3819700/108137963-fa2d4280-708a-11eb-885c-dc8044e8ff8d.jpg)

**Comment by marknoble** [comment]
@mgabdev I just tested the change in my PR and with the spinner issue fixed in Edge (Chromium) users can now navigate to https://dissenter.com/ and log-in, then the Dissenter plugin works in Edge.

![Dissenter working Screenshot 2021-02-16 195751](https://user-images.githubusercontent.com/3819700/108141224-4d09f880-7091-11eb-86a8-e7504e53ba5b.jpg)

It's a workaround, but a very low-impact one that makes the plugin quite usable on Edge now.
